crates/subspace-runtime/src/lib.rs

```diff
@@ -136,7 +136,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: Cow::Borrowed("subspace"),
     impl_name: Cow::Borrowed("subspace"),
     authoring_version: 0,
-    spec_version: 2,
```

Comment:
lets not bump this yet.

I do a see a new problem with maintaing a new branch for taurus @dariolina 
Apart from the spec_versin, there is also sometime storge versions for each pallet that involves migrations. This will problematic when we have a seperate branch for Taurus.

I still think having a single source of truth with same spec_version would be most non-confusing way to go